packages/fxa-auth-server/test/remote/security_events.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { assert } = require('chai');
+const Client = require('../client')();
+const TestServer = require('../test_server');
+
+const config = require('../../config').getProperties();
+
+describe('security events functional test', () => {
+  let server;
+
+  before(function() {
+    this.timeout(15000);
+    config.securityHistory.ipProfiling.allowedRecency = 0;
+    config.signinConfirmation.skipForNewAccounts.enabled = false;
+    return TestServer.start(config).then(s => {
+      server = s;
+    });
+  });
+
+  it('returns securityEvents on creating and login into an acount', () => {
+    const email = server.uniqueEmail();
+    const password = 'abcdef';
+    return Client.create(config.publicUrl, email, password)
```

Comment:
You will need to expose some helper functions on the client api.js that uses a sessionToken

https://github.com/mozilla/fxa/blob/e3d1bdd4c31a487a9ef22a03a604eabf4d930819/packages/fxa-auth-server/test/client/index.js#L503

Reply:
Also see https://github.com/mozilla/fxa/blob/e3d1bdd4c31a487a9ef22a03a604eabf4d930819/packages/fxa-auth-server/test/client/api.js#L770, for where api calls the auth-server.